### DISCUSSION TITLE
Sound on S100102

### DISCUSSION THREAD

**Discussion by Mat-Moo** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#811] [General]
So I have a square S100102 pump 6 wire connected to a pcb V2B - I believe all configured correctly, but struggling to get any sound. Latest firmware (Compiled and uploaded today) - Looking around it could need different voltage? But what do I need to change on the board setup/config
![20241109_175618](https://github.com/user-attachments/assets/a3cebbe7-43d5-44dc-aefb-a6209d844e36)

**Comment by visualapproach** [comment]
Hi! I don't know. It seems there is a different circuit on the newer pumps.
